samples/custom_dispatch/xnnpack/plugin/system_plugin.c

```diff
@@ -44,8 +63,82 @@ typedef struct {
   iree_hal_executable_plugin_allocator_t host_allocator;
   FILE* file;
   pthreadpool_t threadpool;
+  struct cache* cache;
 } system_plugin_t;
 
+inline static void log_debug(const char* format, ...) {
```

Comment:
You can do 
```c
#ifdef XNNPACK_PLUGIN_LOG
#define log_debug(...) fprintf (stderr, __VA_ARGS__)
#else
#define log_debug(...)
#endif  // XNNPACK_PLUGIN_LOG
```

I'd prefer upper cases for macro. It would be nice to have a macro for `cache_dump_key()`.

Reply:
Nice! Much more concise